src/core/client/framework/lib/form/helpers.tsx

```diff
@@ -44,6 +44,7 @@ export const formatPercentage = (v: any) => {
   return Math.round(v * 100).toString();
 };
 
+export const parseBool = (v: any) => !!v;
```

Comment:
Strong suggestion to replace with:

```suggestion
export const parseBool = (v: any) => Boolean(v);
```

Reply:
![image](https://user-images.githubusercontent.com/14221600/54188943-10025800-44b1-11e9-9c11-c42080db9f96.png)
Ok 😃